src/pages/Beta/Stake/DetailModal/EarnedWidget/index.tsx

```diff
@@ -26,15 +25,15 @@ const EarnedWidget: React.FC<Props> = ({ stakingInfo }) => {
 
   const rewardToken = stakingInfo?.rewardToken
   const usdcPriceTmp = useUSDCPrice(rewardToken)
-  const usdcPrice = CHAINS[chainId].is_mainnet ? usdcPriceTmp : undefined
+  const usdcPrice = CHAINS[chainId].mainnet ? usdcPriceTmp : undefined
```

Comment:
just question out of curiosity, why do we have condition based on mainnet flag at many places ? what if user is trying to test on testnet ?

Reply:
Many price calculations of the interface contain divisions by zero which crash the site if the tokens do not have a dollar value. So we decided to remove all dollar values display for testnet since they are useless.